integrations/hardhat/package.json

```diff
@@ -16,15 +16,32 @@
   "main": "./index.js",
   "scripts": {
     "lint": "prettier --cache --check .",
-    "format": "prettier --cache --write ."
+    "format": "prettier --cache --write .",
+    "test": "mocha --exit --recursive 'test/**/*.test.ts'"
   },
   "dependencies": {
     "@oasisprotocol/sapphire-paratime": "workspace:^"
   },
   "peerDependencies": {
-    "hardhat": "^2.0.0"
+    "hardhat": "npm:@oasislabs/hardhat@^2.12.1"
```

Comment:
This can be left alone. You should have `@oasislabs/hardhat` as a strict dependency. Peer dependencies are not automatically installed by several popular package managers, so removing `hardhat` and replacing it with `@oasislabs/hardhat` is extra work.